node/README.md

```diff
@@ -143,6 +148,7 @@ https://docs.env0.com/reference
 You can compile and run the CLI from source by cloning the repo from Github and then running the following:
 ```bash
 # clone the repo from github
+# make sure env0 lib is not installed globally
 yarn install
 yarn link # link your local copy of the CLI to your terminal path
```

Comment:
doesn't it bypasses the globally installed comment? or the terminal first looks at global commands?

Reply:
The terminal gives the global command an higher priority 😭